kubernetes-client/pom.xml

```diff
@@ -209,7 +209,6 @@
     </dependency>
     <dependency>
       <groupId>org.bouncycastle</groupId>
-      <optional>true</optional>
```

Comment:
Not sure we want to make this a required dependency. `bcprov-jdk15on` is resolved as a transitive dependency through this too

Reply:
You are right. I have revert both pom modifications.